main.py

```diff
@@ -1,210 +1,16 @@
-                carAdminPhone
-              }
-              }
-            }
-        """).safe_substitute({'name': self.name, 'hwId': self.hw_id, 'companyName': self.companyName,
-                              'underTest': self.underTest, 'adminPhone': self.adminPhone})
-        return t
-
-    def handle_json_response(self, json_response):
-        # Car adding does not return any errors, always success even if car already exists
-        print(json_response)
-
-
-class StationAdder(Query):
-    def __init__(self, endpoint, login_cookie, name: str, latitude: float, longitude: float, contact_phone):
-        super().__init__(endpoint, login_cookie)
-        self.name = name
-        self.latitude = latitude
-        self.longitude = longitude
-        self.contact_phone = contact_phone
-
-    def get_query(self):
-        return Template("""
-                mutation Q{
-                  StationMutation{addStation(station: {name: "$name", latitude: $latitude, longitude: $longitude,
-                                contactPhone: "$contactPhone"}){
-                    id
-                  }
-                  }
-                }
-            """).safe_substitute({'name': self.name, 'latitude': self.latitude, 'longitude': self.longitude,
-                                  'contactPhone': self.contact_phone})
-
-    def handle_json_response(self, json_response):
-        pass
-
-
-class CarInfoGetter(Query):
-    """Call parent exec() function and then pass result to function get_id_from_json() to get id"""
-    def __init__(self, endpoint, login_cookie, car_name):
-        super().__init__(endpoint, login_cookie)
-        self.car_name = car_name
-
-    def get_query(self) -> str:
-        return Template("""
-            query Q{
-              CarQuery{
-                cars(where: {name: "$name"}){
-                  nodes{
-                    id
-                    name
-                  }
-                }
-              }
-        }
-        """).safe_substitute({'name': self.car_name})
-
-    def handle_json_response(self, json_response):
-        pass
+import json
 
-    def get_id_from_json(self, json_response):
-        """Extracts id from json response"""
-        cars = json_response["data"]["CarQuery"]["cars"]["nodes"]
-        if len(cars) == 0:
-            raise Exception(f"Car with name {self.car_name} doesn't exists!")
-        if len(cars) > 1:
-            raise Exception(f"There is more ({len(cars)}) cars with name {self.car_name}!")
-        return cars[0]["id"]
+from fleet.query.Query import Query
+from fleet.query.Login import get_login_cookie, ENDPOINT
+from fleet.query.Car import CarAdder, CarDeleter
+from fleet.query.User import UserAdder, UserDeleter
+from fleet.query.Station import StationAdder, StationDeleter
+from fleet.query.Route import RouteAdder, RouteDeleter
+from fleet.query.Order import OrderAdder
+from fleet.data.Cookie import Cookie
+from fleet.data.Stop import Stop
 
 
 class AllIdGetter(Query):
```

Comment:
it would probably be better if `AllIdGetter` was moved to separate file `fleet/query/utils.py`